lib/game.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :publisher, :last_played_at
+
+  def initialize(publish_date = 'Undefined', publisher = 'Undefined', last_played_at = 'Undefined', archived: false)
```

Comment:
[Required] Game class initializer is missing the multiplayer variable which is public. Kindly please add a multiplayer variable for your initializer.
[Required] last_played_at is a Date data type but it is initialized as String. Kindly fix this. You can use `last_played_at = Time.now.strftime('%d/%m/%Y')`

Reply:
@darikmohammed can you please be clear on the first required? I did not see a provided variable name in the requirement which is what confuses me. Kindly be explicit.

For the second, I will fix it up, thank you.

Reply:
<img width="203" alt="Screen Shot 2022-09-06 at 11 44 33 PM" src="https://user-images.githubusercontent.com/56404835/188734529-2d17b710-a804-48b2-b1a6-6f3c53044f62.png">
@Olamarx , 
In the Game.rb class you should have the variable multiplayer but it is not declared in your case

Reply:
Oh!!! Now I see, that I was following the requirement on the issue page. The last_play_at was pointed there but not multiplayer.

I will get that done now.

Reply:
Done